R/R/prophet.R

```diff
@@ -1274,37 +1268,203 @@ fit.prophet <- function(m, df, ...) {
 #' forecast <- predict(m, future)
 #' plot(m, forecast)
 #' }
-#'
 #' @export
-predict.prophet <- function(object, df = NULL, ...) {
+predict.prophet <- function(m, df = NULL, ...) {
```

Comment:
In the earlier review I had suggested changing everything below from object to m, but I forgot an important detail: having the argument here be named `object` is actually required by R because we are making an implementation of `predict`. So that will unfortunately have to be changed back.